src/back/glsl/mod.rs

```diff
@@ -1441,6 +1441,21 @@ impl<'a, W: Write> Writer<'a, W> {
                     for sta in case.body.iter() {
                         self.write_stmt(sta, ctx, indent + 2)?;
                     }
+
+                    // Write fallthrough comment if the case is fallthrough,
+                    // otherwise write a break, if the case is not already
+                    // broken out of at the end of its body.
+                    if case.fall_through {
+                        writeln!(self.out, "{}/* fallthrough */", INDENT.repeat(indent + 2))?;
+                    } else if !matches!(
+                        case.body.last(),
+                        Some(&Statement::Break)
```

Comment:
we should probably add a helper method here so that this looks like `statement.is_jump()` or something similar

Reply:
I was considering adding something like that, but I wasn't very sure about what to call it and whether this one use would justify the addition. Since this PR is merged already, should I do a follow up PR for this?